src/ui/ui_get_public_key.c

```diff
@@ -0,0 +1,123 @@
+/*****************************************************************************
+ *   Ledger App Boilerplate.
+ *   (c) 2020 Ledger SAS.
+ *   (c) 2021 COZ Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *****************************************************************************/
+
+#include <stdbool.h>  // bool
+#include <string.h>   // memset
+
+#include "os.h"
+#include "ux.h"
+#include "glyphs.h"
+
+#include "ui_get_public_key.h"
+#include "constants.h"
+#include "../globals.h"
```

Comment:
A bit out of the scope of this PR but... those relative paths are hurting my eyes ; )

Reply:
All gone